mono/utils/mono-threads.h

```diff
@@ -129,22 +129,32 @@ and reduce the number of casts drastically.
 
 
 #if defined (USE_COOP_GC)
-#define USE_COOP_BACKEND
-#define MONO_THREADS_PLATFORM_REQUIRES_UNIFIED_SUSPEND 1
-
+#  define USE_COOP_BACKEND
```

Comment:
This is not how our coding convention works. All preprocessing rules must be anchored to the left.
No spaces between hash and the directive.

What we usually do too is to add comments in the mid/end parts of outer branches to explain the conditions.
```
#if A
#if b
...
#endif
#else /* A */
....
#endif /* A */ 
```